programmers/가운데-글자-가져오기.js

```diff
@@ -0,0 +1,12 @@
+function solution(s) {
+    let answer = ``
+    let evens = s.length / 2
+    let odds = s.length / 2 - 0.5
+    if(s.length % 2 === 0) {
+        answer = s[evens - 1] + s[evens]
+    }
+    else {
+        answer = s[odds]
+    }
+    return answer;
```

Comment:
{} 안에 들어갈 내용이 한줄일 경우 아래와 같이 생략 가능
s.length를 문제와 같이 계속 사용할 경우
변수를 선언하는것이 더욱 좋아보임.
```suggestion
   if(s.length % 2 === 0) answer = s[evens - 1] + s[evens]
    else answer = s[odds]
    return answer;
```